package.json

```diff
@@ -34,5 +40,8 @@
   "browser": {
     "gaia-component": "./bower_components/gaia-component/gaia-component.js",
     "gaia-icons": "./bower_components/gaia-icons/gaia-icons.js"
+  },
+  "dependencies": {
+    "gaia-component": "git://github.com/gaia-components/gaia-component.git"
```

Comment:
Adding this will mean that when running `npm install` both Bower and NPM will install a copy of `gaia-component`, is that intended. This is due to the `"install"` hook we have in `"scripts"` running `bower install`.